src/app/services/system-log.service.ts

```diff
@@ -15,11 +15,12 @@ export class SystemLogService {
   /**
    *  GET | fledge/syslog
    */
-  public getSysLogs(source: String, level: String, limit: Number = 0, offset: Number = 0) {
+  public getSysLogs(source: String, level: String, limit: Number = 0, offset: Number = 0, keyword: string) {
     let params = new HttpParams();
     params = params.set('limit', limit.toString());
     params = params.set('offset', offset.toString());
     params = params.set('nontotals', true);
+    params = params.set('keyword', keyword);
```

Comment:
We should probably add a check if keyword.trim() length  > 0  then params = params.set('keyword', keyword);